documentation/peer-testing-1/heuristic-evaluation-experiment-instructions.md

```diff
@@ -0,0 +1,51 @@
+5. **Error prevention.** Technical problems should not occur during User operations. System provides necessary intructions for users to smoothly operate.Sys
+6. **Recognition rather than recall.** UI designers should minimize the use of user's memory when they are guided to use the system. For example, in some shopping website, there is a recently-viewed list to help users remind what they are interested in.
+7. **Flexibility and efficiency of use.** System should be designed as for all the user groups rather than explicitly assuming that all the users are experienced. For example, for novice users, they can use function through tool bar on the top of the window. However, for experienced users, they can use shortcut keys.
+8. **Aesthetic and minimalist design.** System should minimize the unwanted things and try to grab user's attention on the main function of each interface.
+In short, Designers should build the system as simple as possible.
+9. **Help users recognize, diagnose and recover from errors.** Designers should assume users are unable to understand technical terminology, therefore, error messages should almost always be expressed in plain language to ensure nothing gets lost in translation. For example, when users are trying to log in to the system, there should be a forgot-password link for users to recover their account.
+10. **Help and documentation.** All the basic information such as the introduction and purpose of the software, FAQ section,should be displayed in an interface such as help center. The help page is neccesary even if the system is simple to understand. It helps user experience be better.
+
+## Before entering the evaluation ##
+### Session checklist: ###
+- System is setup and ready to go.
+- Description of the system for participant.
+- Three lists of tasks for mentors, mentees, and Admins.
+- Duration of the tasks should be approximately 40 minutes total.
+- Be ready to document observations during the evaluations.
+- have a list of online questions ready for participants to complete by applying _Google Forms_. Questions can be usability related or open-ended. The Questionnaire should be 5-point Likert scale. 
+- I have created a google form for the questionnaire. Link:https://docs.google.com/forms/d/1K0egKKFqWlHBHL8dpMpvDg4dmS5h3WP_Hp-WnMYRE7Q/prefill.
+
+### Setup your PC: ###
+- Please make sure you **install ZOOM** properly on your PC rather than on a phone or a tablet device before running the sessions. Click the following link to download: https://zoom.us/download.
+- To using remote accessing, make sure you installed the **Chrome Remote Desktop option**. When you are done installing it, a one-time code will be generated for access. You will tell the participants the code for getting access. Click the following link to download:https://remotedesktop.google.com/support/.
```

Comment:
Zoom itself allows for remote control, the Chrome extension isn't needed if Zoom works. My hope is that we can get the whole team to work on Zoom with no need for anyone to use Chrome remote. Can you update to make it clear these are two separate options?

Reply:
Thanks for the clarification! Just did it.